### ISSUE TITLE
Provide missing caption and column titles for tables

### ISSUE THREAD

**Issue by glowcloud** [swagger-api/swagger-ui#10700]
- [ ] provide captions/titles for all tables
- [ ] responses - provide missing column titles when navigating through rows
- [ ] server responses - provide missing column titles when navigating through rows

This task covers issues originally reported in linked sub-issues and:
- https://github.com/swagger-api/swagger-ui/issues/5248
- https://github.com/swagger-api/swagger-ui/issues/7350
